### ISSUE TITLE
Installation via WP-CLI creates plugin folder with .tmp

### ISSUE THREAD

**Issue by morganestes** [mAAdhaTTah/WP-Gistpen#24]
When I install the plugin via `wp-cli`, it creates a folder named `wp-gistpen.tmp` rather than the expected `wp-gistpen`.

```
wp plugin install https://github.com/mAAdhaTTah/WP-Gistpen/releases/download/0.5.0b1/wp-gistpen.zip
```

**Comment by mAAdhaTTah** [comment]
This miiiight be a WP-CLI issue? Do you know any other WordPress plugins that bundle their releases this way?

**Comment by mAAdhaTTah** [comment]
Appears to be a core issue: https://core.trac.wordpress.org/ticket/30945

Looks like there's now a patch for 4.2 to fix this.

**Comment by morganestes** [comment]
Cool beans. I'm watching that ticket on Core now. Thanks for the heads up on it!

**Comment by mAAdhaTTah** [comment]
I'm closing this issue, as this isn't an bug w/ WP-Gistpen.

**Comment by mAAdhaTTah** [comment]
@morganestes [Patch to fix this was accepted for 4.2.](https://core.trac.wordpress.org/ticket/30945)

**Comment by morganestes** [comment]
Sweet! Thanks for keeping and eye on it. I'm looking forward to getting this installed and running on my site soon, just need to get some issues worked out with my host. :(
